prqlc/prqlc/Cargo.toml

```diff
@@ -32,10 +32,21 @@ serde_yaml = [
   "prqlc-parser/serde_yaml",
   "dep:serde_yaml",
 ]
-# Technically tokio could be limited to external tests, but its types are in
-# signatures which would require lots of conditional compilation.
 test-dbs = ["rusqlite", "duckdb", "connector_arrow", "connector_arrow/src_sqlite", "connector_arrow/src_duckdb"]
-test-dbs-external = ["postgres", "mysql", "tiberius", "futures", "tokio", "tokio-util", "connector_arrow/src_mysql", "connector_arrow/src_postgres", "connector_arrow/src_tiberius"]
+test-dbs-external = [
+  # We could attempt to allow running `test-dbs-external` without `test-dbs`,
+  # but it doesn't offer much value, and it's simpler to require both.
+  "test-dbs",
+  "postgres",
+  "mysql",
+  "tiberius",
+  "futures",
+  "tokio",
+  "tokio-util",
+  "connector_arrow/src_mysql",
```

Comment:
FYI @aljazerzen  if you want to have feature names in `connector_arrow` without the `src_`, rust has a way of doing that with `dep:` https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

Reply:
That's a good idea, this naming scheme was inherited from connector-x. I'm even considering splitting the crate, so there would be `connector_arrow`, `connector_arrow_postgres` and so on.

Reply:
OK cool!

> I'm even considering splitting the crate, so there would be `connector_arrow`, `connector_arrow_postgres` and so on.

As a very low information signal — having a single crate with features has the advantage that users can get easy access to lots of DBs quickly. And it lets structs implement common traits etc.